source/server/admin/clusters_handler.h

```diff
@@ -14,6 +14,13 @@
 namespace Envoy {
 namespace Server {
 
+/**
+ * A utility to retrieve admin health status information from a specified host.
```

Comment:
actually, given how this is worded, WDYT of renaming this getHealthFlag and having it return the status?

Maybe that's out of scope for this PR, but would that make more sense?

Reply:
yes, makes sense, can I refactor it as a follow up?

Reply:
sure. WDYT about tightening coverage for this PR?